src/convertToObject.js

```diff
@@ -11,8 +11,21 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  let stringCSS = sourceString.split(';');
```

Comment:
don't reuse one variable, better to create a couple of constants than reassigning one.